pkg/controller/policyrecommendation/policyrecommendation_controller_test.go

```diff
@@ -412,6 +413,40 @@ var _ = Describe("PolicyRecommendation controller tests", func() {
 				err = test.GetResource(c, &tenantBServiceAccount)
 				Expect(err).ShouldNot(HaveOccurred())
 			})
+
+			It("It should not create the trusted bundle config map as it will be created by the tenant controller", func() {
+				// Create the Tenant resources for tenant-a and tenant-b.
```

Comment:
```suggestion
				// Create the Tenant resources
```
There's only one tenant.